packages/fiori/src/UploadCollection.js

```diff
@@ -0,0 +1,354 @@
+		 * <br><br>
+		 * <b>Note:</b> It is up to the application developer to add handler for <code>drop</code> event and handle it.
+		 * <code>ui5-upload-collection</code> only shows an overlay.
+		 *
+		 * @type {boolean}
+		 * @public
+		 */
+		noDnd: {
+			type: Boolean,
+		},
+
+		/**
+		 * Indicates what overlay to show when files are being dragged.
+		 *
+		 * @private
+		 */
+		_dndOverlayMode: {
+			type: String,
+			defaultValue: UploadCollectionDnDOverlayMode.None,
+		},
+	},
+	managedSlots: true,
+	slots: /** @lends sap.ui.webcomponents.fiori.UploadCollection.prototype */ {
+		/**
+		 * Defines the items of the <code>ui5-upload-collection</code>.
+		 * <br><b>Note:</b> Use <code>ui5-upload-collection-item</code> for the intended design.
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			propertyName: "items",
+			type: HTMLElement,
+		},
+
+		/**
+		 * Defines the <code>ui5-upload-collection</code> header.
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		header: {
+			type: HTMLElement,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.fiori.UploadCollection.prototype */ {
+		/**
+		 * Fired when the Delete button of any item is pressed.
+		 * <br><br>
+		 * <b>Note:</b> A Delete button is displayed on each item,
+		 * when the <code>ui5-upload-collection</code> <code>mode</code> property is set to <code>Delete</code>.
+		 * @event
+		 * @param {HTMLElement} item The <code>ui5-upload-collection-item</code> which was renamed.
+		 * @public
+		 */
+		fileDeleted: {
+			detail: {
+				item: { type: HTMLElement },
+			},
+		},
+
+		/**
+		 * Fired when selection is changed by user interaction
+		 * in <code>SingleSelect</code> and <code>MultiSelect</code> modes.
+		 *
+		 * @event
+		 * @param {Array} selectedItems An array of the selected items.
+		 * @public
+		 */
+		selectionChange: {
+			detail: {
+				selectedItems: { type: Array },
+			},
+		},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ * This component allows you to represent files before uploading them to a server, with the help of <code>ui5-upload-collection-item</code>.
+ * It also allows you to show already uploaded files.
+ *
+ * <h3>ES6 Module Import</h3>
+ * <code>import @ui5/webcomponents-fiori/dist/UploadCollection.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.fiori.UploadCollection
+ * @extends UI5Element
+ * @tagname ui5-upload-collection
+ * @appenddocs UploadCollectionItem
+ * @public
+ */
+class UploadCollection extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return UploadCollectionCss;
+	}
+
+	static get template() {
+		return UploadCollectionTemplate;
+	}
+
+	static async onDefine() {
+		await Promise.all([
+			Icon.define(),
+			Label.define(),
+			List.define(),
+			Title.define(),
+			fetchI18nBundle("@ui5/webcomponents"),
```

Comment:
"-fiori"
Each package should use its own bundle. These will be the first translations for this package :) so you get to create it

Reply:
Done